pdc/apps/package/models.py

```diff
@@ -357,6 +359,37 @@ def export(self, fields=None):
         return result
 
 
+class ReleasedFiles(models.Model):
+    file_primary_key = models.IntegerField(blank=False, default=0)
```

Comment:
Looks like Django supports foreign keys to multiple models [1].

Did you try it?

BTW, why is there `default=0`?

Reply:
You mean contenttypes?
I have looked at it for sometime.
Here the contenttypes is about all the tables,  I am doing the research on that

default=0:
As this is not a foreign key, so when do the makemigrations it need a default value.